examples/custom-plugin/plugin/bindings.ts

```diff
@@ -34,8 +34,14 @@ export type RandomNumber = number
 
 import {
 	invoke as TAURI_INVOKE,
-	Channel as TAURI_CHANNEL,
```

Comment:
I think this is reqiured when using channels for them to work. They must be called `Channel` as we basically rely on shadowing to make it work.